### ISSUE TITLE
CapabilityApi and ChannelApi

### ISSUE THREAD

**Issue by bunjix** [denley/courier#24]
Hi, 
Since the last version of play services, the CapabilityApi and ChannelApi has been introduced. 
Are you planning to add them in your library ?

**Comment by denley** [comment]
Yes, I'm definitely planning to support these APIs in Courier. It's a matter of finding some time to go through it thoroughly.

If you have been using either (or both) of these new APIs, I would appreciate if you could show me how you have been using them (perhaps with some code). That would help me to figure out the best way to structure Courier's annotations.
